admin/src/components/Login/Login.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+export default class Login extends React.Component {
```

Comment:
does our linter not enforce specifying proptypes? I feel we should set it too if it is not already.

Reply:
It's not specifying right now, I can add this configuration, perhaps I missed that in the `admin` and is working in `frontend`, but not remember to be enforced to that so far. I will add it, either way, I like that.